### PR TITLE
Fix newly broken tests

### PR DESCRIPTION
**Pull request by erikphansen**
## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: script/run-unit-test.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ const mochaOpts =
 runCommand(
   `LOG_LEVEL=${options[
     'log-level'
-  ].toLowerCase()} ${testRunner} --opts ${mochaOpts} --recursive ${options.path
+  ].toLowerCase()} ${testRunner} --max-old-space-size=4096 --opts ${mochaOpts} --recursive ${options.path
```
Comment (erikphansen) [inline script/run-unit-test.js:48]: This makes `yarn test:unit` run the full suite without running out of RAM

Comment (erikphansen) [reply script/run-unit-test.js:48]: @jbalboni might want to be aware of this

Comment (cvalarida) [reply script/run-unit-test.js:48]: For the record, it looks like this won't affect anything in Jenkins.

**Review by cvalarida — APPROVED** [review]

File: script/run-unit-test.js (hunk 1 above)
Comment (erikphansen) [reply script/run-unit-test.js:48]: Interesting. For now that's fine but it seems like only matter of time before the entire suite no longer runs in Jenkins either.

**Pull request merged**